backend/de.metas.monitoring/src/main/java/de/metas/monitoring/adapter/MicrometerPerformanceMonitoringService.java

```diff
@@ -55,49 +55,37 @@ public <V> V monitor(
 			@NonNull final Callable<V> callable,
 			@NonNull final PerformanceMonitoringService.Metadata metadata)
 	{
+		if(perfMonDataTL == null)
+		{
+			perfMonDataTL = ThreadLocal.withInitial(PerformanceMonitoringData::new);
+		}
```

Comment:
didn't get this one.

my suggesiton was to use `ThreadLocal perfMonDataTL = new ThreadLocal()`